### PR TITLE
chore: add i18n for features page

### LINKED ISSUE #320: 功能特性页面当前英文版不完全
image
参考英文版白皮书 更新功能列表页英文版

### PR DESCRIPTION
**Pull request by juxianzheng**
resolve  #320 
1. update i18n

### CODE REVIEW

File: data/products/api7/features.json [hunk 1]
```diff
@@ -592,39 +592,589 @@
   ],
   "en-US": [
     {
-      "title": "",
+      "title": "API and Service Governanc",
```
Comment (guoqqqi) [inline data/products/api7/features.json:595]: Governanc ??? => Governance

File: data/products/api7/features.json [hunk 2]
```diff
@@ -592,39 +592,589 @@
...
           }
         ]
-      }]
-    },
-    {
-      "title": "",
-      "list": [
+      },
+      {
+        "title": "Finegrained Routing",
```
Comment (guoqqqi) [inline data/products/api7/features.json:663]: Fine-grained Routing

File: data/products/api7/features.json [hunk 3]
```diff
@@ -592,39 +592,589 @@
...
+            "href": ""
+          },
+          {
+            "title": "Priority Matching",
+            "href": ""
+          }
+        ]
+      },
+      {
+        "title": "LoadBalance",
```
Comment (guoqqqi) [inline data/products/api7/features.json:708]:
```suggestion
        "title": "Load-Balance",
```

**Comment by guoqqqi** [comment]
@juxianzheng Please recheck your code for spelling errors

**Comment by guoqqqi** [comment]
@juxianzheng Now it seems that you have cleared out all the content you changed?

**Comment by guoqqqi** [comment]
@juxianzheng  ping

File: data/products/api7/features.json [hunk 4]
```diff
@@ -592,39 +592,589 @@
...
+              "href": ""
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Operations and Maintenance",
+      "list": [
+        {
```
Comment (guoqqqi) [inline data/products/api7/features.json:1110]: image
？？

File: data/products/api7/features.json [hunk 5]
```diff
@@ -592,39 +592,589 @@
...
+          "title": "高效运维",
```
Comment (guoqqqi) [inline data/products/api7/features.json:1111]:
```suggestion
          "title": "CLI",
```

File: data/products/api7/features.json [hunk 6]
```diff
@@ -592,39 +592,589 @@
...
+              "href": ""
+            }, {
+              "title": "Use yaml file to define all rules",
+              "href": ""
+            }, {
+              "title": "Supports unlimited rollback of operations",
+              "href": ""
+            }, {
+              "title": "Easier O&M under k8s",
+              "href": ""
```
Comment (guoqqqi) [inline data/products/api7/features.json:1127]: 重新检查

File: views/Features/Features.tsx [hunk 7]
```diff
@@ -5,7 +5,7 @@ import { NextSeo } from "next-seo";
 
 import { withTranslation } from "../../i18n";
 import { SWrapper, SSection, SMenu, SShowBox } from "./style";
-import features from "../../data/products/api7/features.json";
+import data from "../../data/products/api7/features.json";
```
Comment (guoqqqi) [inline views/Features/Features.tsx:8]: 为啥要换名字，这里应该用 features，更加语义化一些

File: views/Features/Features.tsx [hunk 8]
```diff
@@ -99,7 +99,7 @@ Features.getInitialProps = async (context) => {
   const { lng = "zh-CN" } = (context.req as any) || {};
 
   // After adding the English version replace "zh-CN" with lng in line 84
-  const posts = features["zh-CN"];
+  const posts = data[lng];
```
Comment (guoqqqi) [inline views/Features/Features.tsx:102]: 同样的

File: data/products/api7/features.json [hunk 9]
```diff
@@ -592,39 +592,589 @@
...
+            "href": ""
+          },
+          {
+            "title": "Redis",
+            "href": ""
+          }
+        ]
+      },
+      {
+        "title": "Fault tolerance and downgra ding",
```
[the remark this answers is not in the document]
Comment (guoqqqi) [reply data/products/api7/features.json:804]: ???

File: data/products/api7/features.json [hunk 10]
```diff
@@ -592,39 +592,589 @@
...
+          {
+            "title": "Version Management",
+            "href": ""
+          },
+          {
+            "title": "Release and Abolish API",
+            "href": ""
+          },
+          {
+            "title": "Swagger and OpenAPI",
```
[the remark this answers is not in the document]
Comment (guoqqqi) [reply data/products/api7/features.json:881]: ???

File: data/products/api7/features.json [hunk 11]
```diff
@@ -592,39 +592,589 @@
...
+            "href": ""
+          },
+          {
+            "title": "Generate SDK and documentation",
+            "href": ""
+          }
+        ]
+      },
+      {
+        "title": "Plug-ins Managem ent",
```
[the remark this answers is not in the document]
Comment (guoqqqi) [reply data/products/api7/features.json:891]: ????

File: data/products/api7/features.json [hunk 12]
```diff
@@ -592,39 +592,589 @@
...
         ]
       }
-    ]}
+    ]
+  },
+    {
+      "title": "Security",
+      "list": [
+        {
+          "title": "User Managem ent",
```
[the remark this answers is not in the document]
Comment (guoqqqi) [reply data/products/api7/features.json:913]: ?????

File: views/Features/Features.tsx [hunk 13]
```diff
@@ -99,8 +99,8 @@ Features.getInitialProps = async (context) => {
   const { lng = "zh-CN" } = (context.req as any) || {};
 
   // After adding the English version replace "zh-CN" with lng in line 84
-  const posts = features["zh-CN"];
-
+  const posts = features[lng];
+
```
Comment (guoqqqi) [inline views/Features/Features.tsx:103]:
```suggestion

```

File: data/products/api7/features.json [hunk 14]
```diff
@@ -592,39 +592,589 @@
...
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Operations and Maintenance",
+      "list": [
+        {
+          "title": "CLI",
```
Comment (guoqqqi) [inline data/products/api7/features.json:1111]:
```suggestion
          "title": "Efficient Operations and Maintenance",
```

File: data/products/api7/features.json [hunk 15]
```diff
@@ -592,39 +592,589 @@
...
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Performance and High Availability",
+      "list": [
+        {
+          "title": "QPS",
```
Comment (guoqqqi) [inline data/products/api7/features.json:1051]:
```suggestion
          "title": "Ultimate Performance",
```

File: data/products/api7/features.json (hunk 14 above)
Comment (guoqqqi) [reply data/products/api7/features.json:1111]: also cc @yzeng25, Help check it out when you have time

**Comment by yzeng25** [comment]
Hi @juxianzheng , there are a couple of remaining problems in the top part of this page:

image

1. The first two phrases needs to be translated. 
2. The "Performance and High Availability" does not show up in the beginning section.

File: data/products/api7/features.json [hunk 16]
```diff
@@ -592,39 +592,589 @@
...
+      "title": "Performance and High Availability",
+      "list": [
+        {
+          "title": "Ultimate Performance",
+          "list": [
+            {
+              "title": "Single Core Performance",
+              "href": ""
+            }, {
+              "title": "Minimum latency per request",
```
Comment (yzeng25) [inline data/products/api7/features.json:1057]:
```suggestion
              "title": "0.2 millisecond latency per request",
```

File: data/products/api7/features.json [hunk 17]
```diff
@@ -592,39 +592,589 @@
...
+              "href": ""
+            }
+          ]
+        },
+        {
+          "title": "Deployment",
+          "list": [
+            {
+              "title": "Data plane stateless",
```
Comment (yzeng25) [inline data/products/api7/features.json:1066]:
```suggestion
              "title": "Stateless Data plane",
```

File: data/products/api7/features.json [hunk 18]
```diff
@@ -592,39 +592,589 @@
...
+              "href": ""
+            },
+            {
+              "title": "Automatic cluster selection under multi-tier networks",
+              "href": ""
+            }
+          ]
+        },
+        {
+          "title": "Dynamic and hot updates",
```
Comment (yzeng25) [inline data/products/api7/features.json:1089]: The others are written with all first letter capitalized, might change this to follow the patter
```suggestion
          "title": "Dynamic and Hot Updates",
```

File: data/products/api7/features.json [hunk 19]
```diff
@@ -592,39 +592,589 @@
...
+            {
+              "title": "All changes are hot updated and take effect in real time",
+              "href": ""
+            },
+            {
+              "title": "Plug-in hot updates",
+              "href": ""
+            },
+            {
+              "title": "Hot update of the program itself",
```
Comment (yzeng25) [inline data/products/api7/features.json:1100]:
```suggestion
              "title": " Program hot updates",
```

File: data/products/api7/features.json [hunk 20]
```diff
@@ -592,39 +592,589 @@
...
+      ]
+    },
+    {
+      "title": "Operations and Maintenance",
+      "list": [
+        {
+          "title": "Efficient Operations and Maintenance",
+          "list": [
+            {
+              "title": "Command Line Tools",
```
Comment (yzeng25) [inline data/products/api7/features.json:1114]:
```suggestion
              "title": "Supports Command Line Tools",
```

File: data/products/api7/features.json [hunk 21]
```diff
@@ -592,39 +592,589 @@
...
+              "href": ""
+            }, {
+              "title": "Use REST API for control and easy integration",
```
Comment (yzeng25) [inline data/products/api7/features.json:1117]:
```suggestion
              "title": "Supports REST API for control and easy integration",
```

File: data/products/api7/features.json [hunk 22]
```diff
@@ -592,39 +592,589 @@
...
+              "href": ""
+            }, {
+              "title": "Use yaml file to define all rules",
```
Comment (yzeng25) [inline data/products/api7/features.json:1120]:
```suggestion
              "title": "Uses yaml file to define all rules",
```

File: data/products/api7/features.json [hunk 23]
```diff
@@ -592,39 +592,589 @@
...
+              "href": ""
+            }, {
+              "title": "Supports unlimited rollback of operations",
+              "href": ""
+            }, {
+              "title": "Easier O&M under k8s",
```
Comment (yzeng25) [inline data/products/api7/features.json:1126]:
```suggestion
              "title": "Supports Helm chart, provides easier maintenance in Kubernetes environment",
```

File: data/products/api7/features.json [hunk 24]
```diff
@@ -592,39 +592,589 @@
...
+          {
+            "title": "HTTP Header Matching",
+            "href": ""
+          },
+          {
+            "title": "HTTP Request Method Matching",
+            "href": ""
+          },
+          {
+            "title": "Support for all Nginx variables matching",
```
Comment (yzeng25) [inline data/products/api7/features.json:678]:
```suggestion
            "title": "Supports all Nginx variables matching",
```

File: data/products/api7/features.json [hunk 25]
```diff
@@ -592,39 +592,589 @@
...
+            "href": ""
+          },
+          {
+            "title": "Support for Lua snippets to implement custom matches",
```
Comment (yzeng25) [inline data/products/api7/features.json:682]:
```suggestion
            "title": "Supports Lua snippets for custom matches implementation",
```

File: data/products/api7/features.json [hunk 26]
```diff
@@ -592,39 +592,589 @@
...
+            "href": ""
+          },
+          {
+            "title": "Support for conditional expressions",
```
Comment (yzeng25) [inline data/products/api7/features.json:686]:
```suggestion
            "title": "Supports conditional expressions",
```

File: data/products/api7/features.json [hunk 27]
```diff
@@ -592,39 +592,589 @@
...
+          {
+            "title": "ARM64",
+            "href": ""
+          },
+          {
+            "title": "Kunpeng (certified by Huawei Cloud)",
+            "href": ""
+          },
+          {
+            "title": "AWS, GCP, Ali Cloud, Tencent Cloud and other public clouds",
```
Comment (yzeng25) [inline data/products/api7/features.json:657]:
```suggestion
            "title": "AWS, GCP, Alibaba Cloud, Tencent Cloud and other public clouds",
```

**Comment by guoqqqi** [comment]
@juxianzheng ping

File: public/static/locales/en-US/features.json [hunk 28]
```diff
@@ -0,0 +1,4 @@
+{
+    "title": "APISEVEN Features",
+    "desc": "APISEVEN provides you with a production-ready, seven-tier, full traffic processing platform"
+}
```
Comment (guoqqqi) [inline public/static/locales/en-US/features.json:4]: 文件末尾要有多余的空行

File: public/static/locales/zh-CN/features.json [hunk 29]
```diff
@@ -0,0 +1,4 @@
+{
+    "title": "API7 功能特性",
+    "desc": "API7 为您提供生产可用的七层全流量处理平台"
+}
```
Comment (guoqqqi) [inline public/static/locales/zh-CN/features.json:4]: 同上

File: views/Features/Features.tsx [hunk 30]
```diff
@@ -112,7 +112,7 @@ Features.getInitialProps = async (context) => {
   const { lng = "zh-CN" } = (context.req as any) || {};
 
   // After adding the English version replace "zh-CN" with lng in line 84
```
Comment (guoqqqi) [inline views/Features/Features.tsx:114]:
```suggestion

```
可以移除该注释了

**Comment by yzeng25** [comment]
LGTM, great work @juxianzheng 

cc @guoqqqi

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**